### PR TITLE
Sort by status on mount

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Add sort_by & sort_order to proxy & playground

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2134

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/views/nodes.vue [hunk 1]
```diff
@@ -382,9 +382,13 @@ export default {
             totalSru: convertToBytes(filters.value.minSSD),
             hasGpu: filters.value.gpu || undefined,
             domain: filters.value.gateway || undefined,
+            sortBy: "status",
+            sortOrder: "asc",
           },
           { loadFarm: true },
         );
+        console.log(data);
```
Comment (amiraabouhadid) [inline packages/playground/src/views/nodes.vue:390]: do we really need to console.log this?

File: packages/gridproxy_client/src/builders/nodes.ts [hunk 2]
```diff
@@ -138,6 +144,7 @@ export class NodesBuilder extends AbstractBuilder<NodesQuery> {
       validator: NODES_VALIDATOR,
       queries,
     });
+    console.log("queries", queries);
```
Comment (amiraabouhadid) [inline packages/gridproxy_client/src/builders/nodes.ts:147]: do we need to console.log this?

**Review by AlaaElattar — APPROVED** [review]

File: packages/gridproxy_client/src/builders/nodes.ts [hunk 3]
```diff
@@ -40,6 +40,8 @@ export interface NodesQuery {
   gpuAvailable: boolean;
   ownedBy: number;
   healthy: boolean;
+  sortBy: string;
+  sortOrder: string;
```
Comment (AhmedHanafy725) [inline packages/gridproxy_client/src/builders/nodes.ts:44]: can we make the `sortOrder` enum as it takes only 2 values?

Comment (AhmedHanafy725) [reply packages/gridproxy_client/src/builders/nodes.ts:44]: also, the `sortBy` can be an enum as the sorting queries are well-defined

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**